### DISCUSSION TITLE
Additional feature suggestions for DemoLab

### DISCUSSION THREAD

**Discussion by kpcsvt** [vukasinterzic/Azure#4] [Q&A] [answered]
Hi,

I really like the Demo Lab script. What is the best way to clean up the lab after testing is completed?

Are you planning to implement additional configuration for the lab? Such as installing server roles?

Thank you :)

**Comment by vukasinterzic** [comment, marked as answer]
Hi, thanks for opening the discussion :) 

The best way to clean up is to delete the resource group. Would you like to see some automated cleanup feature implemented?

I am not planning to implement server role configuration. That was already done by someone else, please check the [MSLAB](https://github.com/microsoft/MSLab) repository.

**Reply by kpcsvt** [reply]
Thank you, this is very useful :)
